src/bin/generate_json_schema.rs

```diff
@@ -11,7 +11,11 @@ const ROOT_DIR: &str = env!("CARGO_MANIFEST_DIR");
 fn main() {
     let update_flag = std::env::args().any(|arg| &arg == "--update");
 
-    generate_schema(schema_for!(types::OutputScene), "scene", update_flag);
+    generate_schema(
```

Comment:
It's fine, but in general, it's very bad practice to change formatting in files that you are not touching otherwise. It's increasing difficulty of the review significantlly